test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
cookies yes, but not the form token. For the antiforgery validation to work you need both the cookie and the form token to be present and match.